plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/models/IRunningPod.java

```diff
@@ -0,0 +1,16 @@
+/*******************************************************************************
+ * Copyright (c) 2016 Red Hat, Inc. Distributed under license by Red Hat, Inc.
+ * All rights reserved. This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution, and is
+ * available at http://www.eclipse.org/legal/epl-v10.html
+ * 
+ * Contributors: Red Hat, Inc.
+ ******************************************************************************/
+package org.jboss.tools.openshift.internal.ui.models;
+
+/**
+ * Running pods may be adapted to this interface with Adapters.adapt().
+ */
+public interface IRunningPod {
```

Comment:
naimg feels odd here. I have a IRunningPod but it's not the pod, I have to tell it to give me the pod via #getPod(). To me this type is more of a pod holder or pod provider. Furthermore #getPod wont return the resource but a wrapper, something I dont get when reading the method name #getPod(), feels more like #getPodWrapper, #getPodUIElement or similar.